packages/url-loader/src/plugins/enhance.ts

```diff
@@ -17,8 +17,8 @@ export const EnhancePlugin = plugin({
   props: {
     enhance: true,
   },
-  apply: (cldAsset, options) => {
-    if (options.enhance) {
+  apply: (cldAsset, opts) => {
+    if (opts.enhance) {
```

Comment:
why would this be left in then?

Reply:
Because the type here is a `boolean`, so the value `false` would be legal, so we have to check for that.